<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistEdit</class>
 <widget class="QMainWindow" name="PlaylistEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tano Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../data/logo.qrc">
    <normaloff>:/logo/48x48/logo-editor.png</normaloff>:/logo/48x48/logo-editor.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="labelName">
      <property name="text">
       <string>Playlist name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editName"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelCount">
      <property name="text">
       <string>Channels number:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLCDNumber" name="number">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>25</height>
       </size>
      </property>
      <property name="numDigits">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="PlaylistWidget" name="playlist" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionImport"/>
   <addaction name="actionExport"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionUp"/>
   <addaction name="actionDown"/>
   <addaction name="separator"/>
   <addaction name="actionClose"/>
  </widget>
  <widget class="QDockWidget" name="editWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>329</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>524287</height>
    </size>
   </property>
   <property name="windowIcon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/edit.png</normaloff>:/icons/24x24/edit.png</iconset>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Editor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="editWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelNumber">
       <property name="text">
        <string>Number:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelChannelName">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelCategories">
       <property name="text">
        <string>Categories:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelLanguage">
       <property name="text">
        <string>Language:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelEpg">
       <property name="text">
        <string>EPG:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editNumber">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="maxLength">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="editChannelName"/>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="editCategories"/>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QLineEdit" name="editLanguage"/>
     </item>
     <item row="7" column="1" colspan="2">
      <widget class="QLineEdit" name="editEpg"/>
     </item>
     <item row="12" column="0" colspan="3">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelUrl">
       <property name="text">
        <string>Url:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="editUrl"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="buttonApplyNum">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="../../../data/icons.qrc">
         <normaloff>:/icons/24x24/apply.png</normaloff>:/icons/24x24/apply.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="3">
      <widget class="QLabel" name="labelCategoriesInfo">
       <property name="text">
        <string>&lt;i&gt;You can separate categories with comma.&lt;br&gt;Only first one is saved when exporting to Sagem Js playlist format.&lt;/i&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="3">
      <widget class="QLabel" name="labelNumberInfo">
       <property name="text">
        <string>&lt;i&gt;You need to apply the number.&lt;br&gt;All other values are automatically changed.&lt;/i&gt;</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="labelInfo">
       <property name="text">
        <string>&lt;b&gt;&lt;i&gt;Instructions:&lt;/i&gt;&lt;/b&gt;</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="3">
      <widget class="QLabel" name="labelSeparator">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="updateWidget">
   <property name="windowIcon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/refresh.png</normaloff>:/icons/24x24/refresh.png</iconset>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Update Playlist</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="updateWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="labelFrom">
       <property name="text">
        <string>IP Range:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelTimeout">
       <property name="text">
        <string>Timeout:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="4">
      <widget class="QPushButton" name="buttonUpdate">
       <property name="text">
        <string>Scan and update</string>
       </property>
       <property name="icon">
        <iconset resource="../../../data/icons.qrc">
         <normaloff>:/icons/24x24/refresh.png</normaloff>:/icons/24x24/refresh.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLineEdit" name="ipFrom">
       <property name="inputMask">
        <string notr="true">000.000.000;_</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="ipTimeout">
       <property name="suffix">
        <string> ms</string>
       </property>
       <property name="minimum">
        <number>500</number>
       </property>
       <property name="maximum">
        <number>2000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>800</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPort">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="ipPort">
       <property name="minimum">
        <number>80</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>5000</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QProgressBar" name="progressBar">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QLabel" name="labelUpdate">
       <property name="text">
        <string>&lt;i&gt;This will interrupt your current playback.&lt;/i&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/add.png</normaloff>:/icons/24x24/add.png</iconset>
   </property>
   <property name="text">
    <string>Add channel</string>
   </property>
   <property name="toolTip">
    <string>Add channel</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/remove.png</normaloff>:/icons/24x24/remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete channel</string>
   </property>
   <property name="toolTip">
    <string>Delete channel</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/save.png</normaloff>:/icons/24x24/save.png</iconset>
   </property>
   <property name="text">
    <string>Save and Close</string>
   </property>
   <property name="toolTip">
    <string>Save and Close</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/close.png</normaloff>:/icons/24x24/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/import.png</normaloff>:/icons/24x24/import.png</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionUp">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/go-up.png</normaloff>:/icons/24x24/go-up.png</iconset>
   </property>
   <property name="text">
    <string>Move up</string>
   </property>
   <property name="toolTip">
    <string>Move up</string>
   </property>
  </action>
  <action name="actionDown">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/go-down.png</normaloff>:/icons/24x24/go-down.png</iconset>
   </property>
   <property name="text">
    <string>Move down</string>
   </property>
   <property name="toolTip">
    <string>Move down</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/print.png</normaloff>:/icons/24x24/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/export.png</normaloff>:/icons/24x24/export.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionImportTanoOld">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/file-old.png</normaloff>:/icons/24x24/file-old.png</iconset>
   </property>
   <property name="text">
    <string>Tano Old Playlist</string>
   </property>
  </action>
  <action name="actionImportJs">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/file-js.png</normaloff>:/icons/24x24/file-js.png</iconset>
   </property>
   <property name="text">
    <string>Sagem JS</string>
   </property>
  </action>
  <action name="actionExportJs">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/file-js.png</normaloff>:/icons/24x24/file-js.png</iconset>
   </property>
   <property name="text">
    <string>Sagem JS</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/open.png</normaloff>:/icons/24x24/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/information.png</normaloff>:/icons/24x24/information.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/new.png</normaloff>:/icons/24x24/new.png</iconset>
   </property>
   <property name="text">
    <string>New playlist</string>
   </property>
  </action>
  <action name="actionExportM3UClean">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/file-m3u.png</normaloff>:/icons/24x24/file-m3u.png</iconset>
   </property>
   <property name="text">
    <string>M3U (Original)</string>
   </property>
  </action>
  <action name="actionImportCSV">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/file-csv.png</normaloff>:/icons/24x24/file-csv.png</iconset>
   </property>
   <property name="text">
    <string>CSV</string>
   </property>
   <property name="toolTip">
    <string>CSV</string>
   </property>
  </action>
  <action name="actionExportCSV">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/file-csv.png</normaloff>:/icons/24x24/file-csv.png</iconset>
   </property>
   <property name="text">
    <string>CSV</string>
   </property>
   <property name="toolTip">
    <string>CSV</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/update_alert.png</normaloff>:/icons/24x24/update_alert.png</iconset>
   </property>
   <property name="text">
    <string>Update Information</string>
   </property>
   <property name="toolTip">
    <string>Update Information</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PlaylistWidget</class>
   <extends>QWidget</extends>
   <header>ui/playlist/PlaylistWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../data/icons.qrc"/>
  <include location="../../../data/logo.qrc"/>
 </resources>
 <connections/>
</ui>
